underpants.go

```diff
@@ -257,6 +277,31 @@ func serveHttpProxy(d *disp, w http.ResponseWriter, r *http.Request) {
     return
   }
 
+  if d.config.HasGroups() {
+    authorized := false
+
+AuthCheck:
```

Comment:
This tripped me up and made me think the `break` would jump back up here. Can you move this under the `for`s or is it a style thing?